realm/realm-library/src/main/cpp/io_realm_internal_Collection.cpp

```diff
@@ -433,3 +433,19 @@ Java_io_realm_internal_Collection_nativeGetMode(JNIEnv *env, jclass, jlong nativ
     return -1; // Invalid mode value
 }
 
+JNIEXPORT jlong JNICALL
+Java_io_realm_internal_Collection_nativeCreateResultsFromBacklinks(JNIEnv *env, jclass, jlong shared_realm_ptr, jlong row_ptr, jlong src_table_ptr, jlong src_col_index) {
+    TR_ENTER_PTR(row_ptr)
+    Row* row = ROW(row_ptr);
+    if (!ROW_VALID(env, row))
```

Comment:
Curly braces (I have changed all in #4280) but we have no way to enforce it easily.